CHANGELOG.md

```diff
@@ -54,6 +54,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Removed
 - Remove support for resume task. [#266](https://github.com/greenbone/ospd/pull/266)
 
+[20.08]: https://github.com/greenbone/ospd/compare/ospd-20.08...master
```

Comment:
Hmm not sure which diff to use here. Maybe between old release branch and new release branch?
```suggestion
[20.08]: https://github.com/greenbone/ospd/compare/ospd-2.0...ospd-20.08
```